robocop/run.py

```diff
@@ -70,8 +70,9 @@ def report(self, msg):
             return
         for report in self.reports:
             report.add_message(msg)
-        self.log_message(source=msg.source, line=msg.line, col=msg.col, severity=msg.severity.value,
-                         msg_id=msg.msg_id, desc=msg.desc)
+        self.log_message(source=msg.source, line=msg.line, col=msg.col,
```

Comment:
Maybe in such cases lets go with one line one kwarg? I wonder what looks better

Reply:
We can try but then we will have code stretched vertically so that less code would fit in one screen height.

Reply:
Ok, I split arguments to separate lines.